.github/actions/vercel/action.yaml

```diff
@@ -0,0 +1,106 @@
+name: "VERCEL BUILD AND DEPLOY"
+description: "Builds and deploy vercel project"
+
+inputs:
+  vercel-token:
+    description: "Vercel token"
+    required: true
+  vercel-org-id:
+    description: "Vercel Organization ID"
+    required: true
+  vercel-project-id:
+    description: "Vercel Project ID"
+    required: true
+  ref-name:
+    description: "Branch"
+    required: true
+  sha:
+    description: "Sha"
+    required: true
+  environment:
+    description: "Sha"
+    required: true
+
+outputs:
+  domain:
+    description: "Domain"
+    value: ${{ steps.deploy.outputs.domain }}
+  inspect-url:
+    description: "Inspect URL"
+    value: ${{ steps.deploy.outputs.inspect-url }}
+  alias:
+    description: "Alias"
+    value: ${{ steps.alias.outputs.value }}
+
+runs:
+  using: "composite"
+  steps:
+    - id: branch
+      run: |
+        CLEAN_NAME="${GITHUB_REF_NAME/.staging/}"
+        CLEAN_NAME=$( echo "${CLEAN_NAME}" | sed 's/[^a-zA-Z0-9_-]//g' | tr A-Z a-z | tr _ - | sed 's/-\{2,\}/-/g' )
+        echo "value=${CLEAN_NAME}" >> $GITHUB_OUTPUT
+      shell: bash
+
+    - id: short_sha
+      run: |
+        SHORT_SHA=$( echo "value=$(echo ${{ github.sha }} | cut -c1-7)" )
```

Comment:
You sure we will not need to switch to full she again?

Reply:
Now not used at all

Reply:
Added for the possible future use